### PR TITLE
adjustments event 2019

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: app/components/map.js [hunk 1]
```diff
@@ -192,7 +167,7 @@ function getEventsMapContent(locations) {
                 longitude: maxDistanceTeamLocation.longitude,
             }}/>;
     });
-    const teamPolylines = locations.map(team => {
+    const teamPolylines = props.locations.map(team => {
```
Comment (nerdsupremacist) [inline app/components/map.js:170]: do we need these since they're basically the event starting point?

File: app/components/map.js [hunk 2]
```diff
@@ -256,25 +220,26 @@ export class Map extends Component {
...
         return (
             <View style={styles.container}>
                 <MapView
                     style={styles.container}
                     customMapStyle={mapStyle}
                     initialRegion={{
-                        latitude: 48.5690923,
-                        longitude: 7.6920547,
+                        latitude: 48.150676,
+                        longitude: 11.580984,
```
Comment (nerdsupremacist) [inline app/components/map.js:238]: Hahahaha What?!?!?! This was seriously hard coded?

File: app/components/posting-list.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ export default class PostingList extends React.PureComponent {
     }
 
     renderPosting(row) {
-        return <Posting addLike={this.props.addLike} {...(row.item)} />;
+        return <Posting addLike={this.props.addLike} {...(Object.assign(this.props.navigation, row.item))} />;
```
Comment (nerdsupremacist) [inline app/components/posting-list.js]: isn't it basically 
```suggestion
        return <Posting addLike={this.props.addLike} { ...this.props.navigation, ...row.item } />;
```

**Comment by nerdsupremacist** [comment]
lgtm just have some questions, but no real concerns

File: app/components/map.js (hunk 1 above)
Comment (thepiwo) [reply app/components/map.js:170]: these are all the polylines foreach team

File: app/components/map.js (hunk 2 above)
Comment (thepiwo) [reply app/components/map.js:238]: its the "region" of the map, I did center it the last time to be somewhere between Germany and Barcelona, not it focuses Germany

File: app/components/posting-list.js (hunk 3 above)
Comment (thepiwo) [reply app/components/posting-list.js]: didn't work... as it merges with eventually existing props from React.PureComponent I assume

**Review by nerdsupremacist — APPROVED** [review]

**Pull request merged**